cloudflare/resource_cloudflare_zone_settings_override_test.go

```diff
@@ -62,6 +62,8 @@ func TestAccCloudflareZoneSettingsOverride_Full(t *testing.T) {
 						name, "settings.0.zero_rtt", "off"),
 					resource.TestCheckResourceAttr(
 						name, "settings.0.universal_ssl", "off"),
+					resource.TestCheckResourceAttr(
```

Comment:
This test is failing due to the setting not being persisted to state.

```
=== RUN   TestAccCloudflareZoneSettingsOverride_Full
    testing.go:705: Step 1 error: Check failed: Check 8/8 error: cloudflare_zone_settings_override.test: Attribute 'settings.0.ciphers' not found
--- FAIL: TestAccCloudflareZoneSettingsOverride_Full (21.01s)
FAIL
FAIL	github.com/cloudflare/terraform-provider-cloudflare/cloudflare	21.386s
FAIL
```

Check out the README for running these integration tests if you haven't already. 

Instead of looking for specific values (which I don't think `TypeList` handles too well), just check the number of elements. This line would become `resource.TestCheckResourceAttr(name, "settings.0.ciphers.#", "2")` however if you swap it to that, you'll see it's empty without any elements too.

Reply:
mmm now it fails with:

```
Error: error from makeRequest: HTTP status 400: content "{\"success\":false,\"errors\":[{\"code\":1023,\"message\":\"Advanced Certificate Manager is required to set custom cipher suites\"}],\"messages\":[],\"result\":null}"
```
seems it requires some extra configuration tobe enabled...